src/main/java/org/jenkinsci/plugins/buildtriggerbadge/IconFinder.java

```diff
@@ -79,6 +79,7 @@ private String internalFindForClass(Class clazz)
 		defineIconForCause("com.cloudbees.plugins.flow.FlowCause", "flow-cause.png");
 		defineIconForCause("com.cloudbees.jenkins.plugins.BitBucketPushCause", "bitbucket.png");
 		defineIconForCause("hudson.plugins.git.GitStatus$CommitHookCause", "git-hook-cause.png");
+		defineIconForCause("org.jenkinsci.plugins.urltrigger", "url-trigger-cause.png");
```

Comment:
Hi! thanks for the PR

Your addition seems to be pointing to a package instead of a _Cause_ implementation. Which is unlikely to work.

Just read the code of the URLTrigger plugin, IIUC what you may want instead is the org.jenkinsci.plugins.urltrigger.URLTriggerCause class.